src/utils/sortResults.ts

```diff
@@ -44,23 +45,29 @@ const normalize = (value: number): -1 | 0 | 1 => {
   return 0;
 };
 
+export const comparePosition = (left: IPosition, right: IPosition): -1 | 0 | 1 => {
```

Comment:
This is actually unrelated to the gist of this PR. I initially started with a range based comparison approach. That proved to be a bad idea. However, while building the now defunct `rangeEquals`, I refactored that piece of code.

Although quite a minor change, I decided to keep it along (as it cleans the code a little) and covered it with tests.

Reply:
@P0lip I eventually dropped it from here and promoted it into its own PR (#967)